app/logic/Calendar/OutgoingActions.ts

```diff
@@ -0,0 +1,45 @@
+import type { Event } from "./Event";
+import type { Participant } from "./Participant";
+import { ResponseType, type iCalMethod } from "./Invitation";
+import type { MailAccount } from "../Mail/MailAccount";
+import type { PersonUID } from "../Abstract/PersonUID";
+
+export default class OutgoingActions {
```

Comment:
Can you please rename this to `OutgoingInvitation` and move it to app/logic/Calendar/Invitation/OutgoingInvitation.ts ?

Please also separate the `ResponseType` enum into `IncomingInvitationResponse` and `OutgoingInvitationResponse`.

Reply:
Wherever possible, separate the status into incoming and outgoing, and put them into IncomingInvitation.ts and `OutgoingInvitation.ts`

For the rest, move app/logic/Calendar/Invitation.ts to app/logic/Calendar/Invitation/InvitationStatus.ts